### PR TITLE
Add&remove

### PR DESCRIPTION
**Pull request by LOctavio**
In this milestone, i added functions to add and remove tasks from the to-do list

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -49,44 +63,47 @@ const printList = () => {
...
 const addExamples = () => {
   addTask('Buy eggs', 1);
+  printTask('Buy eggs', 1);
   addTask('Wash the dishes', 2);
+  printTask('Wash the dishes', 2);
   addTask('Feed cats', 3);
+  printTask('Feed cats', 3);
 };
 
 const getLocalStorage = () => {
```
Comment (carlos-ssh) [inline src/index.js:109]: **Kindly solve this issues: **
- I tried to refresh the page and i loose all my tasks in the list. If you check the console in your browser you will find an error related into your line 112.

File: src/deleteTasks.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    item.remove();
+  }
+  tasksList = tasksList2;
+  document.querySelector('ul').innerHTML = '';
+  printList();
+}
+
+export function clearCompleted() {
+  for (let i = 0; i < localStorage.length; i += 1) {
+    const { completed } = JSON.parse(localStorage.getItem(localStorage.key(i)));
```
Comment (carlos-ssh) [inline src/deleteTasks.js]: - After tried to remove complete tasks i have an error. Kindly solve it.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @LOctavio, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Davidosky007** [comment]
Hi @LOctavio, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**